src/main/java/net/wurstclient/hacks/AutoStealHack.java

```diff
@@ -41,6 +51,11 @@ public boolean areButtonsVisible()
 	
 	public long getDelay()
 	{
-		return delay.getValueI();
+		return delay.getValueI() + getRandom();
+	}
+
+	private long getRandom()
+	{
+		return random.nextInt(randomDelay.getValueI()) * 2 - (randomDelay.getValueI());

```

Comment:
I think this should be something like `random.nextInt(randomDelay.getValueI() * 2 + 1) - randomDelay.getValueI()`. My version produces an integer from `randomDelay.getValueI()` to `randomDelay.getValueI()` inclusive.